packages/cloud_firestore/cloud_firestore_web/lib/src/collection_reference_web.dart

```diff
@@ -0,0 +1,193 @@
+// Copyright 2017, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_interface.dart';
+import 'package:firebase/firestore.dart' as web;
+import 'package:meta/meta.dart';
+
+import 'package:cloud_firestore_web/src/document_reference_web.dart';
+import 'package:cloud_firestore_web/src/query_web.dart';
+
+/// Web implementation for Firestore [CollectionReferencePlatform]
+class CollectionReferenceWeb extends CollectionReferencePlatform {
+  /// instance of Firestore from the web plugin
+  final web.Firestore webFirestore;
+  final FirestorePlatform _firestorePlatform;
+  final List<String> pathComponents;
```

Comment:
Can this be hidden?

Reply:
This can't be hidden. It's from `CollectionReferencePlatform`.

Reply:
(This has been hidden) :P

Reply:
It's great that `webFirestore` is now hidden, but `pathComponents` is still here and I'm wondering if it can be removed.

`CollectionReferencePlatform` extends `QueryPlatform` which [already has a `pathComponents` member](https://github.com/FirebaseExtended/flutterfire/blob/master/packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/platform_interface/query.dart#L50) that should be getting assigned in the call to `super()`. If we're not overriding it in any meaningful way then it's confusing to repeat it here.